### ISSUE TITLE
adjustHeadElementsIndentation() shouldn't flatten the indentation of nested elements

### ISSUE THREAD

**Issue by buo** [negibouze/html-webpack-pug-plugin#9]
Here is my code snippet:

```pug
// layout.pug
head
    meta(charset="utf-8")
    title
        block title
    link(rel="stylesheet" href="style.css")
    script(src="script.js")
body
    block body
```

I have title block in head element so that I can set the title in child template as follows:

```pug
// index.pug
extends layout

block title
    | Welcome

block body
    h1 Welcome
```

But `adjustHeadElementsIndentation()` flattens the indentation of head elements and generates unwanted result:

```pug
// generated layout.pug
head
    meta(charset="utf-8")
    title
    block title
    link(rel="stylesheet" href="style.css")
    script(src="script.js")
body
    block body
```

**Comment by buo** [comment]
Another examples are inline style and script.

```pug
// before
head
    style(type="text/css").
        body {
            font-size: 13px;
        }
    script(type="text/javascript").
        var MY_VAR = 'foobar';
```

```pug
// after
head
    style(type="text/css").
    body {
    font-size: 13px;
    }
    script(type="text/javascript").
    var MY_VAR = 'foobar';
```

**Comment by negibouze** [comment]
Thank you for your report.

**Comment by negibouze** [comment]
I released a new version (v0.2.0).

**Comment by buo** [comment]
It works very well. Thank you 👍
